src/components/SuggestToWeb.js

```diff
@@ -0,0 +1,221 @@
+import React, { useState, useEffect, useRef } from "react";
```

Comment:
There's a lot of comments in this file and the changes are fairly complicated, this might be a good one to rewrite together via pair programming